app/pages/patient/patientinfo.js

```diff
@@ -468,36 +444,35 @@ var PatientInfo = translate()(React.createClass({
         </div>
       </div>
     )
-  },
+  }
 
-  isSamePersonUserAndPatient: function() {
+  isSamePersonUserAndPatient() {
     return personUtils.isSame(this.props.user, this.props.patient);
-  },
+  }
 
-  isEditingAllowed: function(permissions) {
+  isEditingAllowed(permissions) {
     if (_.isPlainObject(permissions)) {
-      return permissions.hasOwnProperty('custodian') || permissions.hasOwnProperty('root');
+      return Object.prototype.hasOwnProperty.call(permissions,'custodian') || Object.prototype.hasOwnProperty.call(permissions,'root');
     }
 
     return false;
-  },
+  }
 
-  getDisplayName: function(patient) {
+  getDisplayName(patient) {
     return personUtils.patientFullName(patient);
-  },
+  }
 
-  getAgeText: function(patient, currentDate) {
-    const { t } = this.props;
-    var patientInfo = personUtils.patientInfo(patient) || {};
-    var birthday = patientInfo.birthday;
+  getAgeText(patient, currentDate) {
+    const patientInfo = personUtils.patientInfo(patient) || {};
+    const birthday = patientInfo.birthday;
 
     if (!birthday) {
       return;
```

Comment:
This function return a string normally, the return statement should return something.
```suggestion
      return null;
```